celery/app/defaults.py

```diff
@@ -78,6 +78,7 @@ def __repr__(self):
         scheduler=Option('celery.beat:PersistentScheduler'),
         schedule_filename=Option('celerybeat-schedule'),
         sync_every=Option(0, type='int'),
+        cron_starting_deadline=Option(None, type=int)
```

Comment:
Should be the `'int'` string instead of the `int`class

Reply:
Thanks for noticing